spec/models/user_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
Please add some unit tests for the `User` model

Reply:
They are not on this branch @diraulo, we are waiting for this PR to be merged in order to proceed implementing the signup function. This is just a setup branch..

Reply:
The thing is the moment you add the devise gem to your application and do the following as per devise's documentation:

- setup a user model
- add devise routes `devise_for: users`
- etc.

That's practically all it takes to have every authentication actions setup already (Sign up, Sign in, etc...)

If you check your routes you'll see that you have routes for all of them, if you fire up your application server, you can sign-up and sign-in (all the forms are already created for you by devise)

The user model should be fairly straightforward to test, if you check how we tested previous models, and look at the schema file, you'll be able to see what fields you should check for in your unit tests.